### ISSUE TITLE
missing cgroups options in containers.podman.podman_container

### ISSUE THREAD

**Issue by inntran** [containers/ansible-podman-collections#259]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

"cgroups" at containers.podman.podman_container does not accept options other than "default" and "disabled".
But the podman-run's manpage provides more than that.

**Steps to reproduce the issue:**

1. Create a playbook, include this task:
```
containers.podman.podman_container:
  cgroups: no-conmon
```

**Describe the results you received:**
`FAILED! => {"changed": false, "msg": "value of cgroups must be one of: default, disabled, got: no-conmon"}`

**Describe the results you expected:**
```
changed: [192.168.1.2]
```
**Additional information you deem important (e.g. issue happens only occasionally):**
http://docs.podman.io/en/latest/markdown/podman-run.1.html
```
--cgroups=enabled|disabled|no-conmon|split
Determines whether the container will create CGroups.

Default is enabled.

The enabled option will create a new cgroup under the cgroup-parent. 
The disabled option will force the container to not create CGroups, and thus conflicts with CGroup options (--cgroupns and --cgroup-parent). 
The no-conmon option disables a new CGroup only for the conmon process. 
The split option splits the current CGroup in two sub-cgroups: one for conmon and one for the container payload. It is not possible to set --cgroup-parent with split.
```

**Version of the `containers.podman` collection:** 1.5.0
**Either git commit if installed from git: `git show --summary`**
**Or version from `ansible-galaxy` if installed from galaxy: `ansible-galaxy collection list | grep containers.podman`**

```
(paste your output here)
```

**Output of `ansible --version`:**

```
ansible-3 2.9.21
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/yinsong/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.6/site-packages/ansible
  executable location = /bin/ansible-3
  python version = 3.6.8 (default, Aug 13 2020, 07:46:32) [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
```

**Output of `podman version`:**

```
Version:      3.0.2-dev
API Version:  3.0.0
Go Version:   go1.15.7
Built:        Wed Apr  7 08:36:54 2021
OS/Arch:      linux/amd64

```

**Output of `podman info --debug`:**

```
(paste your output here)
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
(paste your output here)
```

**Playbok you run with ansible (e.g. content of `playbook.yaml`):**

```
(paste your output here)
```

**Command line and output of ansible run with high verbosity**

**Please NOTE: if you submit a bug about idempotency, run the playbook with `--diff` option, like:**

`ansible-playbook -i inventory --diff -vv playbook.yml`

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by lowne** [comment]
I have the same issue. The relevant line is [here](https://github.com/containers/ansible-podman-collections/blob/6f7c1f340fa44cdfd03c90921558d43f6d24d477/plugins/module_utils/podman/podman_container_lib.py#L25), but the options themselves were [added in podman 1.80](https://github.com/containers/podman/blob/490915c68e6fce48508d42d138ac1c92c877abe7/changelog.txt#L3072) and [2.10](https://github.com/containers/podman/blob/490915c68e6fce48508d42d138ac1c92c877abe7/changelog.txt#L1783) - I don't know if further checks are required.

**Comment by lowne** [comment]
It seems that `--tz=` is also missing, and was similarly added in [2.10](https://github.com/containers/podman/blob/490915c68e6fce48508d42d138ac1c92c877abe7/changelog.txt#L1752)

**Comment by sshnaidm** [comment]
@lowne please create a separate issue about `--tz`, maybe other parameters that you found as well.

**Comment by sshnaidm** [comment]
Fixed in #262
